recipes-graphics/wayland/weston-init.bbappend

```diff
@@ -24,11 +24,12 @@ INI_UNCOMMENT_ASSIGNMENTS:append:mx8mq-nxp-bsp = " \
 # FIXME: The 8QM and 8QXP SoCs have better performance without G2D so don't enable it
 # Ideally, this should be seamless and Vivante ought to handle it internally and take the fastest
 # rendering code.
+INI_UNCOMMENT_USE_G2D               ?= ""
 INI_UNCOMMENT_USE_G2D:imxgpu2d      ?= "use-g2d=1"
 INI_UNCOMMENT_USE_G2D:mx8qm-nxp-bsp  = ""
 INI_UNCOMMENT_USE_G2D:mx8qxp-nxp-bsp = ""
 INI_UNCOMMENT_USE_G2D:mx8dx-nxp-bsp  = ""
-INI_UNCOMMENT_ASSIGNMENTS:append:imxgpu2d = " \
+INI_UNCOMMENT_ASSIGNMENTS:append = " \
```

Comment:
I'm unclear about this. So far we don't add `imxgpu2d` for i.MX 93 https://github.com/nxp-imx/meta-imx/blob/kirkstone-5.15.71-2.2.0/meta-bsp/conf/machine/include/imx-base-extend.inc#L32. The part has 2D, though I think it is different from other 2D implementaitons we support.

Reply:
I am confused now. The imx93 has a different 2D GPU?

Reply:
i.MX93 has no GPU hardware, but a different PXP G2D hardware.

Reply:
@junzhuimx thanks for the clarification.